doc/manual/integers-and-floating-point-numbers.rst

```diff
@@ -412,14 +412,23 @@ types:
 
 .. doctest::
 
-    julia> (typemin(Float16),typemax(Float16))
-    (-Inf16,Inf16)
```

Comment:
I think somehow the function we use to show a tuple changed.

```jl
julia> show(Inf16)
Inf16
julia> print(Inf16)
Inf
```

IIRC this also affects the precision of floating point numbers in tuple. (there should be at least one other broken doctest due to this)